examples/dynamo/eg7/alg.x90

```diff
@@ -1,3 +1,38 @@
+! -----------------------------------------------------------------------------
```

Comment:
Again, I think this is my fault but this file should have the MO copyright notice first as I must have lifted it from LFRic.

Reply:
Done.